src/router/metadata/cap.rest.security.template.ts

```diff
@@ -20,7 +20,7 @@ export default function makeSecurity(authConfig: Auth) {
                 },
             ],
         };
-        if (authConfig.strategy.oauthUrl) {
+        if (authConfig.strategy.oauth?.oauthAuthorizationUrl) {

```

Comment:
For this line we just need to ensure that the oauth object is present - if it is then we know `oauthAuthorizationUrl` will be there as it is required
```suggestion
        if (authConfig.strategy.oauth) {

```